dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPJava17Test.java

```diff
@@ -0,0 +1,159 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.jsp23.fat.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collections;
+import java.util.logging.Logger;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.jsp23.fat.JSPUtils;
+import com.meterware.httpunit.GetMethodWebRequest;
+import com.meterware.httpunit.WebConversation;
+import com.meterware.httpunit.WebRequest;
+import com.meterware.httpunit.WebResponse;
+
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.annotation.MaximumJavaLevel;
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+/**
+ * JSP 2.3 tests which use Java 17 specific features.
+ *
+ * Tests must only run when Java 17 or later is in use.
+ *
+ */
+@MinimumJavaLevel(javaLevel = 17)
```

Comment:
The java 11 and Java 21 tests are in full mode and the 21 and 8 tests are in lite mode, is that intentional?

Reply:
17 and 8 are LITE mode while the 21 and 17 are FULL.

I kept 8 the same, but decided to only make one of the newer tests run in LITE mode.  We can't build with gradle running 21 ( overall requires more setup to run a FAT w/ 21), so I opted for Java 17 to be LITE.

Reply:
ok sounds good!